test/endtoend/wdio.remote.conf.js

```diff
@@ -15,21 +17,21 @@ exports.config = {
   logLevel: 'warn',
 
   capabilities: [
-    { browserName: 'firefox', version: 'latest', platform: 'Windows 10' },
-    { browserName: 'firefox', version: 'latest-1', platform: 'Windows 10' },
-    { browserName: 'chrome', version: 'latest', platform: 'Windows 10' },
-    { browserName: 'chrome', version: 'latest-1', platform: 'Windows 10' },
-    // { browserName: 'safari', version: 'latest', platform: 'macOS 11' },
-    // { browserName: 'safari', version: '13.1', platform: 'macOS 10.15' },
-    // { browserName: 'Safari', platformName: 'iOS', platformVersion: '12.2', deviceOrientation: 'portrait', deviceName: 'iPhone 8 Simulator' },
-    { browserName: 'chrome', platformName: 'Android', platformVersion: '6.0', deviceOrientation: 'portrait', deviceName: 'Android Emulator' },
+    { browserName: 'firefox', browserVersion: 'latest', platformName: 'Windows 10' },
+    { browserName: 'firefox', browserVersion: 'latest-1', platformName: 'Windows 10' },
+    { browserName: 'chrome', browserVersion: 'latest', platformName: 'Windows 10' },
+    { browserName: 'chrome', browserVersion: 'latest-1', platformName: 'Windows 10' },
+    { browserName: 'safari', browserVersion: 'latest', platformName: 'macOS 11' },
+    { browserName: 'safari', browserVersion: '13.1', platformName: 'macOS 10.15' },
+    { browserName: 'Safari', 'appium:deviceName': 'iPhone 12 Simulator', 'appium:deviceOrientation': 'portrait', 'appium:platformVersion': '14.3', platformName:'iOS' },
+    // { browserName: 'Chrome', 'appium:deviceName': 'Android GoogleAPI Emulator', 'appium:deviceOrientation': 'portrait', 'appium:platformVersion': '11.0', platformName: 'Android' },
```

Comment:
Have you tried with other versions? I guess you have, but if not, this is what I got from the configurator:

```
caps = {};
caps['appiumVersion'] = '1.20.2';
caps['deviceName'] = 'Android Emulator';
caps['deviceOrientation'] = 'portrait';
caps['browserName'] = 'Chrome';
caps['platformVersion'] = '8.0';
caps['platformName'] = 'Android';
```

Reply:
I did try this one, and didn't get more green: https://github.com/transloadit/uppy/actions/runs/1041251713. Anyway, we are listing only the last two Android versions in our browserlist, so it would be fragile to test 8.0 in our end2end suite (we were testing on 6.0 and it stopped working all of the sudden because we started using features unsupported as of chromium 67.0).
I don't understand why it's failing, Sauce Labs docs hardly refer to web app testing with Appium, all of their examples are for app testing... I'm giving it up, if you want to pick it up, be my guest :)

Reply:
Right, ok, fair enough, just making sure we tried what we could :)

Reply:
No worries, sorry my comment sounds a bit passive-aggressive. Let me rephrase that: I'm a bit frustrated with Sauce Labs docs, I'm probably missing something to make the Android emulator work and I'd prefer if we kept Android as part of the e2e suite. I feel me spending more time on this wouldn't be productive, but maybe a pair of fresh eyes would be able to find the missing piece. I'd prefer if it didn't block this PR to get back to a green CI ASAP.

Reply:
No worries, thanks for the clarification :) I agree, and I share the frustration, I set up the end to end tests a few years ago and pretty much stayed away since. Green light to the PR, no need to block over Android. Thanks!